File: forge/src/main/java/juuxel/adorn/platform/forge/client/ClientNetworkBridgeForge.java
package juuxel.adorn.platform.forge.client;

import juuxel.adorn.client.ClientNetworkBridge;
import net.minecraft.network.protocol.common.custom.CustomPacketPayload;
import net.neoforged.neoforge.client.network.ClientPacketDistributor;

public final class ClientNetworkBridgeForge implements ClientNetworkBridge {
    @Override
    public void sendToServer(CustomPacketPayload payload) {
        ClientPacketDistributor.sendToServer(payload);
    }
}

File: forge/src/main/kotlin/juuxel/adorn/platform/forge/client/ClientNetworkBridgeForge.kt
package juuxel.adorn.platform.forge.client

import juuxel.adorn.client.ClientNetworkBridge
import juuxel.adorn.platform.forge.networking.SetTradeStackC2SMessage
import net.minecraft.item.ItemStack
import net.neoforged.neoforge.network.PacketDistributor

class ClientNetworkBridgeForge : ClientNetworkBridge {
    override fun sendSetTradeStack(syncId: Int, slotId: Int, stack: ItemStack) {
        PacketDistributor.SERVER.noArg().send(SetTradeStackC2SMessage(syncId, slotId, stack))
    }
}
